src/base64url.ts

```diff
@@ -1,12 +1,16 @@
-import { base64url } from 'jose';
+import { Base64 } from 'js-base64';
 
-const encode = (input: string | Uint8Array): string => base64url.encode(input);
+const encode = (input: string): string => Base64.encodeURI(input);
 
-const decode = (input: string | Uint8Array): string => {
-  return new TextDecoder().decode(base64url.decode(input));
+const decode = (input: string): string => {
+  return Base64.decode(input);
 };
```

Comment:
That's right. But this was created for consistency in case anyone gets confused with base64.

I think it can be reduced a bit by not wrapping it in a `Base64Url` object. Please check the change and tell me what you think :) @berendsliedrecht

Reply:
Fix looks good!